From 7638f379cac3869abdd8530faa09cc7481e99ec6 Mon Sep 17 00:00:00 2001
From: James Booth <boothj5@gmail.com>
Date: Wed, 4 Mar 2015 23:32:47 +0000
Subject: [PATCH] Fixed tests for --disable-otr

---
 Makefile.am       | 6 +++++-
 tests/testsuite.c | 2 ++
 2 files changed, 7 insertions(+), 1 deletion(-)

diff --git a/Makefile.am b/Makefile.am
index d9fa972..809c3ee 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -44,6 +44,7 @@ tests_sources = \
 	src/roster_list.c src/roster_list.h \
 	src/xmpp/xmpp.h src/xmpp/form.c \
 	src/ui/ui.h \
+	src/otr/otr.h \
 	src/command/command.h src/command/command.c \
 	src/command/commands.h src/command/commands.c \
 	src/tools/parser.c \
@@ -63,7 +64,6 @@ tests_sources = \
 	src/ui/titlebar.h src/ui/statusbar.h src/ui/inputwin.h \
 	src/server_events.c src/server_events.h \
 	tests/xmpp/stub_xmpp.c \
-	tests/otr/stub_otr.c \
 	tests/ui/stub_ui.c \
 	tests/log/stub_log.c \
 	tests/config/stub_accounts.c \
@@ -104,6 +104,9 @@ otr3_sources = \
 otr4_sources = \
 	src/otr/otrlib.h src/otr/otrlibv4.c src/otr/otr.h src/otr/otr.c
 
+otr_test_sources = \
+	tests/otr/stub_otr.c
+
 themes_sources = themes/*
 
 script_sources = bootstrap.sh configure-debug install-all.sh
@@ -111,6 +114,7 @@ script_sources = bootstrap.sh configure-debug install-all.sh
 man_sources = docs/profanity.1
 
 if BUILD_OTR
+tests_sources += $(otr_test_sources)
 if BUILD_OTR3
 core_sources += $(otr3_sources)
 endif
diff --git a/tests/testsuite.c b/tests/testsuite.c
index ddfb45c..645a1a4 100644
--- a/tests/testsuite.c
+++ b/tests/testsuite.c
@@ -346,9 +346,11 @@ int main(int argc, char* argv[]) {
         unit_test(cmd_account_set_eval_password_when_password_set),
         unit_test(cmd_account_set_muc_sets_muc),
         unit_test(cmd_account_set_nick_sets_nick),
+#ifdef HAVE_LIBOTR
         unit_test(cmd_account_show_message_for_missing_otr_policy),
         unit_test(cmd_account_show_message_for_invalid_otr_policy),
         unit_test(cmd_account_set_otr_sets_otr),
+#endif
         unit_test(cmd_account_set_status_shows_message_when_invalid_status),
         unit_test(cmd_account_set_status_sets_status_when_valid),
         unit_test(cmd_account_set_status_sets_status_when_last),
-- 
2.3.1

